assets/scss/templates/_signup.scss

```diff
@@ -48,18 +49,36 @@
       font-family: $dp-font-family-proximanova;
     }
 
-    .signup-form {
-      .holder-small {
-        display: inline-block;
-        width: 47.4%;
+    .content-legal {
+      margin-top: 30px;
+    }
 
-        &:nth-child(1) {
-          margin-right: 4%;
+    .holder {
+      &-group {
+        display: flex;
+        flex-wrap: wrap;
+        flex-direction: column;
+      }
+
+      &-size1of1 {
+        flex: 1;
+
+        &:nth-last-child(-n+3) {
+          margin-top: 30px;
         }
       }
 
-      ul > li:nth-child(n+3) {
-        margin-top: 20px;
+      &-col {
+        flex-wrap: nowrap;
+        flex-direction: row;
+      }
+
+      &-size1of2 {
+        flex: 1;
+
+        &:nth-child(1) {
+          margin-right: 4%;
```

Comment:
Strange measure warning!

Reply:
what is the problem?

Reply:
@IgnacioRodrigues seems to be a magic number. It would be much better if we use an absolute unit here 👌

Reply:
Ok!